### PR TITLE
Update js-interop.md

### PR DESCRIPTION
**Pull request by RicoTrevisan**
Small update on the example code to make it a tad more obvious and more like the other examples on the page.

### CODE REVIEW

File: guides/client/js-interop.md [hunk 1]
```diff
@@ -253,10 +253,15 @@ and the return value is ignored.
 For example, the following option could be used to guarantee that some attributes set on the client-side are kept intact:
 
 ```javascript
-onBeforeElUpdated(from, to){
-  for (const attr of from.attributes){
-    if (attr.name.startsWith("data-js-")){
-      to.setAttribute(attr.name, attr.value);
+...
+let liveSocket = new LiveSocket("/live", Socket, {
+  params: { _csrf_token: csrfToken },
```
Comment (rhcarvalho) [inline guides/client/js-interop.md:258]: To be consistent with the code base:

```suggestion
  params: {_csrf_token: csrfToken},
```

File: guides/client/js-interop.md [hunk 2]
```diff
@@ -253,10 +253,15 @@ and the return value is ignored.
...
+  hooks: Hooks,
+  onBeforeElUpdated(from, to) {
+    for (const attr of from.attributes) {
+      if (attr.name.startsWith("data-js-")) {
+        to.setAttribute(attr.name, attr.value);
+      }
```
Comment (rhcarvalho) [inline guides/client/js-interop.md:264]: As per the earlier paragraph in the docs, the correct snippet would be to set a `dom` option with the callback:

```suggestion
  dom: {
    onBeforeElUpdated(from, to) {
      for (const attr of from.attributes) {
        if (attr.name.startsWith("data-js-")) {
          to.setAttribute(attr.name, attr.value);
        }
      }
```

---

See examples in the official test suite:

- https://github.com/phoenixframework/phoenix_live_view/blob/6648f774354e308efc26d7e44e6e68d5a1b12fe1/assets/test/view_test.js#L918-L924
- https://github.com/phoenixframework/phoenix_live_view/blob/6648f774354e308efc26d7e44e6e68d5a1b12fe1/test/support/e2e/form_feedback.ex#L25-L58

**Review by rhcarvalho — COMMENTED** [review]
:purple_heart: 

Thanks! Adding more context to that example makes a lot of sense, especially since it is easy to miss that the callback needs to go within the `dom` option.

**Comment by SteffenDE** [comment]
Thank you both! 🙌

**Pull request merged**